src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -35,15 +36,8 @@
 public class BiomeManager
 {
     private static TrackedList<BiomeEntry>[] biomes = setupBiomes();
-
-    public static List<Biome> oceanBiomes = new ArrayList<Biome>();
-
-    static
-    {
-        oceanBiomes.add(Biomes.OCEAN);
-        oceanBiomes.add(Biomes.DEEP_OCEAN);
-        oceanBiomes.add(Biomes.FROZEN_OCEAN);
-    }
+    @Deprecated //Don't call this method directly, use the proper methods in this class
+    public static List<OceanBiomeEntry>[] oceanBiomes = setupOceanBiomes();
```

Comment:
This is still a binary breaking change as you have changed the type of the field.

Reply:
True.
In that case, the best option would be to deprecate the old field and make a new field called `forgeOceanBiomes`(or some other similar name) to handle the ocean biomes, correct?
(I've mentioned this before, but in 1.13+ outside of this PR, the old field isn't even called outside of `BiomeManager` in forge)

Reply:
Yes, make a new (private) field and deprecate the old one.
It doesn't matter if it's used or not, the point is it is public and so someone _could_ use it, so it has to stay.